src/SprykerEco/Yves/Heidelpay/Controller/CreditCardController.php

```diff
@@ -157,7 +158,7 @@ protected function hydrateCreditCardRegistrationToQuote(
      */
     protected function redirectToSummaryStep(): RedirectResponse
     {
-        $summaryStepUrl = $this->getConfig()->getYvesCheckoutSummaryStepUrl();
+        $summaryStepUrl = $this->getFactory()->getConfig()->getYvesCheckoutSummaryStepUrl();
```

Comment:
do we really need this `getFactory` interlayer?
controller resolves config as well
same for similar cases